schwab.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Chetan Narsude <chetan+csv2txf@celeral.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implements Schwab
+
+Charles Schwab gain/loss output provides already-reconciled transactions, i.e.,
+each buy/sell pair comes in a single record, on a single line.
+
+Does not handle:
+* dividends
+* short sales
+* partial lot sales
+"""
+
+import csv
```

Comment:
Please add above:

```
from __future__ import annotations
```

This will make it easier for you to add type annotations below.